### DISCUSSION TITLE
Specify HTML class for a TD

### DISCUSSION THREAD

**Discussion by jfuentesa** [orchidsoftware/platform#2607] [Q&A]
Hello:

I have this method in a Screen:

```
public function layout(): iterable
{
    return [
       Layout::table('notifications', [
           TD::make('author', __("Author")),
           TD::make('message', __("Message")),
           TD::make('created_at', __("Date"))
                ->render(fn ($post) => date("d-m-Y H:i", strtotime($post->created_at)) ),
        ])
    ];
}
```

Is it possible to specify a HTML class="" for the TDs?

I tried this method, via an inner SPAN, but the x-orchid-icon is not parsed and I guess it makes things more complicated:

```
    public function layout(): iterable
    {
        return [
            Layout::table('notifications', [
                TD::make('author', __("Author")),
                TD::make('message', __("Message")),
                TD::make('created_at', __("Date"))
                    ->render(fn ($post) => date("d-m-Y H:i", strtotime($post->created_at)) ),
                TD::make('action-icons', "")
                    ->render(fn ($post) => '<span class="no-wrap"><a href="edit-post?id=' . $post["post_id"] . '><x-orchid-icon path="pencil" /></a></span>'),
            ])
        ];
    }
```

Thank you.

**Comment by tabuna** [comment]
`render` - Returns any string that should be rendered as is.

If you want to use Blade, you can return a template:

```php
TD::make()->render(fn () => view('path.to.view'))
```

If the s[tring is short](https://laravel.com/docs/10.x/blade#rendering-inline-blade-templates), then you can use:
```php
use Illuminate\Support\Facades\Blade;

TD::make()->render(fn (string $name) => Blade::render('Hello, {{ $name }}', ['name' => 'Julian Bashir']))
```

There are other ways, in addition to those indicated, for example, the use of components. You can read it [here](https://orchid.software/en/docs/table/#components).

**Reply by jfuentesa** [reply]
Hello @tabuna, thank you for your reply.

Unfortunately I didn't explain myself very well... The question is, how can I use a html class in the html td generated by TD::make()?

I mean:
`<td class="my-class">`

But I can use the span with a class like you said, thank you anyway.

Thank you.
